src/main/anovos/data_analyzer/association_evaluator.py

```diff
@@ -187,11 +186,6 @@ def variable_clustering(
     sample_size
         Maximum sample size (in terms of number of rows) taken for the computation.
         Sample dataset is extracted using random sampling. (Default value = 100000)
-    stats_unique
```

Comment:
By removing stat_unique, you will also need to remove from the workflow.py for `variable_clustering` as well. Remove `stat_unique` in here should solve the issue in workflow test: https://github.com/anovos/anovos/blob/3d4c3d08ec4b868e182b4543831644a7a0ff2e3c/src/main/anovos/workflow.py#L77